homeassistant/components/monarchmoney/sensor.py

```diff
@@ -0,0 +1,198 @@
+"""Sensor config - monarch money."""
+
+from collections.abc import Callable
+from dataclasses import dataclass
+from datetime import datetime
+from typing import Any
+
+from typedmonarchmoney.models import MonarchAccount, MonarchCashflowSummary
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import CURRENCY_DOLLAR, PERCENTAGE, EntityCategory
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+
+from . import MonarchMoneyConfigEntry
+from .entity import MonarchMoneyAccountEntity, MonarchMoneyCashFlowEntity
+
+
+@dataclass(frozen=True, kw_only=True)
+class MonarchMoneySensorEntityDescription(SensorEntityDescription):
+    """Describe a sensor entity."""
+
+    value_fn: Callable[[MonarchAccount], StateType | datetime] | None = None
+    summary_fn: Callable[[MonarchCashflowSummary], StateType] | None = None
+    picture_fn: Callable[[Any], str] | None = None
```

Comment:
Please split the entity descriptions

Reply:
You can then also remove the nullability
```suggestion
@dataclass(frozen=True, kw_only=True)
class MonarchMoneySensorEntityDescription(SensorEntityDescription):
    """Describe a sensor entity."""

    value_fn: Callable[[MonarchAccount], StateType | datetime]
    summary_fn: Callable[[MonarchCashflowSummary], StateType] | None = None
    picture_fn: Callable[[Any], str] | None = None
```
That can help you remove the null checks